src/Models/User.php

```diff
@@ -53,6 +55,10 @@ class User extends Model implements AuthenticatableContract, AuthorizableContrac
         "name",
         "email",
         "password",
+        "avatar_path",
+    ];
+    protected $attributes = [
+        "avatar_path" => Constants::SPEAKER_DEFAULT_AVATAR_PATH,
```

Comment:
```suggestion
        "avatar_path" => Constants::USER_DEFAULT_AVATAR_PATH,
```
I think it should be User avatar